tests/tools/test_sa.py

```diff
@@ -1,7 +1,14 @@
 import numpy as np
 import pytest
 import statsmodels
-from lifelines import CoxPHFitter, KaplanMeierFitter
+from lifelines import (
```

Comment:
Why import the lifelines versions? You should always use the ehrapy implementation